src/Components/NewBlog.js

```diff
@@ -1,14 +1,16 @@
 import axios from "axios";
 import { useState } from "react";
 
-const CreatePost = () => {
+export const postApiUrl = "https://blogpostapi1.herokuapp.com/"
+
+const NewBlog = () => {
   const [post, setPost] = useState({
     name: "",
     title: "",
     content: "",
   });
 
-  const output = (param) => {
+  const handlePost = (param) => {
```

Comment:
`handleChange` would be a better name.

And you still need to change the function signature, you can get `name` from `event.target`